common/regex.js

```diff
@@ -13,10 +13,10 @@ module.exports = {
     TYPE: r(`I type ${string}${elInEl}`),
     REPLACE: r(`I replace the contents${elInEl} with ${string}`),
     OPEN: r(`I open${page}`),
-    WAIT_FOR_RESULTS: r(`I wait for results to load`),
+    WAIT_FOR_ROUTE: r(`I wait for ${string} to resolve`),
```

Comment:
oh this a good one. Let's ADD this as an option instead of replacing the old one for backwards compatibility